2022-11-13 20:36:13,075 INFO    MainThread:34483 [wandb_setup.py:_flush():71] setting env: {}
2022-11-13 20:36:13,075 INFO    MainThread:34483 [wandb_setup.py:_flush():71] setting login settings: {}
2022-11-13 20:36:13,075 INFO    MainThread:34483 [wandb_init.py:_log_setup():371] Logging user logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/mappov1/wandb/run-20221113_203613-1t0qptj8/logs/debug.log
2022-11-13 20:36:13,075 INFO    MainThread:34483 [wandb_init.py:_log_setup():372] Logging internal logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/mappov1/wandb/run-20221113_203613-1t0qptj8/logs/debug-internal.log
2022-11-13 20:36:13,076 INFO    MainThread:34483 [wandb_init.py:init():404] calling init triggers
2022-11-13 20:36:13,076 INFO    MainThread:34483 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-11-13 20:36:13,076 INFO    MainThread:34483 [wandb_init.py:init():460] starting backend
2022-11-13 20:36:13,076 INFO    MainThread:34483 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-13 20:36:13,092 INFO    MainThread:34483 [backend.py:ensure_launched():216] starting backend process...
2022-11-13 20:36:13,097 INFO    MainThread:34483 [backend.py:ensure_launched():221] started backend process with pid: 34523
2022-11-13 20:36:13,098 INFO    MainThread:34483 [wandb_init.py:init():469] backend started and connected
2022-11-13 20:36:13,101 INFO    MainThread:34483 [wandb_init.py:init():533] updated telemetry
2022-11-13 20:36:13,117 INFO    MainThread:34483 [wandb_init.py:init():563] communicating current version
2022-11-13 20:36:14,224 INFO    MainThread:34483 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-13 20:36:14,224 INFO    MainThread:34483 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-11-13 20:36:14,372 INFO    MainThread:34483 [wandb_init.py:init():606] starting run threads in backend
2022-11-13 20:36:15,857 WARNING MainThread:34483 [wandb_init.py:init():952] interrupted
Traceback (most recent call last):
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 931, in init
    run = wi.init()
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 611, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 423, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/josyula/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/josyula/miniconda3/lib/python3.9/threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "/home/josyula/miniconda3/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
